samples/msal-node-samples/ElectronSystemBrowserTestApp/src/index.html

```diff
@@ -2,12 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
-    <title>MSAL Node Electron Sample App</title>
-    
-    <!-- adding Bootstrap 4 for UI components  -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
-    <link rel="SHORTCUT ICON" href="https://c.s-microsoft.com/favicon.ico?v2" type="image/x-icon">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no"> 
```

Comment:
We should define a [content security policy](https://www.electronjs.org/docs/latest/tutorial/security#7-define-a-content-security-policy) in these pages e.g. https://github.com/Azure-Samples/ms-identity-javascript-nodejs-desktop/blob/main/App/index.html

Reply:
Done